src/main/java/fi/evident/gradle/beanstalk/DeployTask.java

```diff
@@ -38,6 +38,8 @@ protected String getVersionLabel() {
         if (versionLabel.endsWith("-SNAPSHOT")) {
             String timeLabel = new SimpleDateFormat("yyyyMMdd'.'HHmmss").format(new Date());
             versionLabel = versionLabel.replace("SNAPSHOT", timeLabel); // Append time to get unique version label
+        } else {
+            versionLabel = versionLabel + deployment.getVersionSuffix();
         }
         return deployment.getVersionPrefix() + versionLabel;
```

Comment:
It's a bit surprising that `versionPrefix` and `versionSuffix` are not symmetric. Now `versionPrefix` is always prepended, whereas `versionSuffix` is not appended for snapshot-versions. Is this by design or by accident?

What if the code here was the less surprising:

```java
return deployment.getVersionPrefix() + versionLabel + deployment.getVersionSuffix();
```

Would this fit your use case?

Reply:
Yeah this would work too!!  I just figured people that were using snapshot would want that to be at the end instead of a custom suffix.  I'm ok with either approach.